warehouse/bigquery/bigquery.go

```diff
@@ -222,13 +231,13 @@ func partitionedTable(tableName string, partitionDate string) string {
 	return fmt.Sprintf(`%s$%v`, tableName, strings.ReplaceAll(partitionDate, "-", ""))
 }
 
-func (bq *HandleT) loadTable(tableName string, forceLoad bool, getLoadFileLocFromTableUploads bool) (partitionDate string, err error) {
+func (bq *HandleT) loadTable(tableName string, forceLoad bool, getLoadFileLocFromTableUploads bool, skipTempTableDelete bool) (partitionDate string, stagingTableName string, err error) {
```

Comment:
make return as struct ? returning three outputs might not be so readable sometimes

Reply:
done